src/hooks/transactions/useSignTransactionsWithDevice.tsx

```diff
@@ -150,7 +154,7 @@ export function useSignTransactionsWithDevice({
             sessionId,
             status: TransactionBatchStatusesEnum.signed,
             transactions: Object.values(newSignedTransactions).map((tx) =>
-              parseTransactionAfterSigning(tx as Transaction, isLedger)
+              parseTransactionAfterSigning(tx)
```

Comment:
what about the isLedger flag? That should probably not be dropped

Reply:
Please see my comment [here](https://github.com/ElrondNetwork/dapp-core/pull/236/files#r882595889)